docs/user_guide.md

```diff
@@ -477,16 +477,68 @@ optional arguments:
   -s, --steps           Print steps.
 ~~~~~~~~~~~~~~~~~~~~~
 
-## 4. checkers mode:
+## 4. `checkers` mode
 
-List all available checkers.
+List the checkers available in the installed analyzers which can be used when
+performing an analysis.
 
-The ```+``` (or ```-```) sign before a name of a checker shows whether the checker is enabled (or disabled) by default.
+### Old `checkers` command (before `v6.0` release)
+
+**WARNING!** Usage of this command is deprecated and with the release of `v6.0`
+will be superseded by the new command, see below.
+
+The ```+``` (or ```-```) sign before a name of a checker shows whether the
+checker is enabled (or disabled) by default.
 
 ~~~~~~~~~~~~~~~~~~~~~
 CodeChecker checkers
 ~~~~~~~~~~~~~~~~~~~~~
 
+### New `checkers` command (since `v5.8` release)
+
+By default, `codechecker-checkers` will list only the *enabled* checkers, one
```

Comment:
As I've already mentioned by in one of my comments we should list all the checkers by default, not just the enabled ones.